docs/changes.md

```diff
@@ -1,7 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
-
+- `12.7` Multiple Updates:
```

Comment:
Happy for this to be classed as a patch (which means you don't need to update the changelog) ... I have applied the Small Tweak/Improvement classification to the PR.